File: lib/src/main/java/com/tuti/api/data/DueAmount.java
package com.tuti.api.data;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class DueAmount implements Serializable {
    public String getDueAmount() {
        return dueAmount;
    }

    public String getTotalAmount() {
        return totalAmount;
    }

    public String getMinAmount() {
        return minAmount;
    }

    public String getPaidAmount() {
        return paidAmount;
    }

    @SerializedName("due_amount")
    String dueAmount;

    @SerializedName("total_amount")
    String totalAmount;

    @SerializedName("min_amount")
    String minAmount;

    @SerializedName("paid_amount")
    String paidAmount;
}

File: lib/src/main/java/com/tuti/api/data/DueAmount.kt
package com.tuti.api.data

import kotlinx.serialization.SerialName

@kotlinx.serialization.Serializable
data class DueAmount (
    @SerialName("due_amount")
    val dueAmount: String = "",

    @SerialName("total_amount")
    val totalAmount: String = "",

    @SerialName("min_amount")
    val minAmount: String = "",

    @SerialName("paid_amount")
    val paidAmount: String = "",
)